### ISSUE TITLE
Console spam for Minecraft 1.16.4 (Forge 35.1.13) Servers

### ISSUE THREAD

**Issue by MarioMario24680** [mactso/HarderFishesAndAmphibians#1]
Hello, I just wanted to let you know about a small issue with the current version of your mod, version 1.1.0.5. Apparently, every time a new River Guardian spawns, they leave a message in the console, but because the River Guardians spawn so frequently, this can, in turn, flood the console occasionally. Do you think you could tweak the mod so that these messages don't show up anymore? I do like having a lot of River Guardians around and would prefer not to fix this issue by just lowering their spawn rate in the config file. Here's a picture of the same message I keep getting over and over again:
https://prnt.sc/wa0no8

On another note, I might have a recommendation for a configuration option for their AI. Could it be possible to make the River Guardians not attack Villagers or Villager-based modded mobs? You see, I have a fisherman village right next to an Ocean biome that's constantly being harassed by the Guardians, and I was hoping that the Villagers could avoid being killed.

**Comment by mactso** [comment]
Absolutely.  I'll fix this and get a new version without the debugging messages.

Hmm.  River guardians are only supposed to attack child mobs (including child villagers) near the water.   If they are attacking adults, then that's a bug.   Also river guardians spawn deep in the ocean and shallow in the river.  Is there a river biome near there?  Anyway, stopping villager attacks is simple.   I'll default to no villager attacks of any kind for now  and then restore configurable child villager attacks in the next major release with the Gurty.

And feel free to lower the spawn rate.  That's what it is there for.  Actual spawn rates vary *immensely* depending on what other mods are loaded and what their spawn weight values are.   What's reasonable with one set of mods is unreasonable with another set of mods.  

Anyway, I'll probably have a fix for this out late tonight or sometime tomorrow.

**Comment by MarioMario24680** [comment]
Thank you so much. If it's possible, could you also include compatibility with the Guard Villagers mod?

**Comment by mactso** [comment]
Release 6
Stops river guardians from attacking Villagers
Stops river guardians from announcing spawn location unless debugging level is 1 or higher.
Adds option to turn off river guardian attacks on villager children who get too close to the water.

What kind of compatibility did you need in Guard Villagers mod?

**Comment by MarioMario24680** [comment]
I know that the River Guardians attack the Guard Villagers (who are essentially adult Villagers with weapons), so if there'd be a configurable option to turn that off, set to "off" by default, that would be nice.

**Comment by mactso** [comment]
Oh, they use non-standard entities.   They really are not villagers in the code.  There isn't an easy way to check for them since they are non-standard.   I'll have to find a way that works whether or not that mod is installed.
